api/sheets/get_property.md

```diff
@@ -6,41 +6,55 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the values of a given property for a node.
+# Retrieve property values for nodes
```

Comment:
Should we use Retrieve for the page titles as well?

Reply:
I think I mentioned this in another PR, but I'm not sure if the change would negatively affect the user experience on the sidebar due to the increased width.

Reply:
+1